app/routes/review.py

```diff
@@ -295,6 +291,7 @@ def get_summary_reviews():
   return {"data": get_summary_reviews_helper(request.args.get('beach_id'))}
 
 @bp.route("/patch", methods=["PATCH"])
+@jwt_required()
```

Comment:
if we add JWT auth we should also ensure that the user owns the review that they are modifying